### ISSUE TITLE
[Bug]: Pipeline imported failed from towhee on OS ubuntu 

### ISSUE THREAD

**Issue by binbinlv** [towhee-io/towhee#307]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Pipeline imported failed from towhee on OS ubuntu 

```
>>> from towhee import pipeline
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/__init__.py", line 22, in <module>
    from towhee.engine.engine import Engine, start_engine
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/engine.py", line 18, in <module>
    from towhee.engine.pipeline import Pipeline
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/pipeline.py", line 17, in <module>
    from towhee.engine.graph_context import GraphContext
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/graph_context.py", line 21, in <module>
    from towhee.engine.operator_context import OperatorContext, OpStatus
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/operator_context.py", line 19, in <module>
    from towhee.engine.operator_runner.runner_base import RunnerStatus
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/operator_runner/__init__.py", line 22, in <module>
    from towhee.engine.operator_io.reader import DataFrameReader
  File "/home/zong/binbin/towhee/towhee-test-env/lib/python3.6/site-packages/towhee/engine/operator_io/__init__.py", line 16, in <module>
    import towhee.engine.operator_io.reader as io_reader
AttributeError: module 'towhee.engine' has no attribute 'operator_io'
```

### Expected Behavior

pipeline imported from towhee  successfully

### Steps To Reproduce

```python
from towhee import pipeline
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93): 0.2.3.dev3
- OS(Ubuntu or CentOS): Ubuntu 18.04
- CPU/Memory: 
- GPU: 
- Others:
```

**Comment by Ashvith** [comment]
I believe that this is with respect to the **main** branch, because the pip version works fine. Would you mind sharing how I can install the **main** branch locally?

**Comment by binbinlv** [comment]
@Ashvith 

It is an issue when using python3.6, and now it is fixed by pr #311.

And verified, it is fixed now. Thanks.
